deploy/operator.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     spec:
       containers:
       - name: portworx-operator
-        image: portworx/px-operator:1.11.0-dev
+        image: portworx/px-operator:23.4.1-dev
```

Comment:
hmm.. we haven't changed this at all (even 23.5.1 branch still has the old operator version)

Reply:
do we want to keep 1.11.0-dev?

Reply:
It may be harmless, but I'd still roll it back  (perhaps fix it with 23.7.0)?
[ it's not a good time for experiments right now.. ]

Reply:
+1 , no experiments now.

Reply:
reverted back to 1.11.0-dev https://github.com/libopenstorage/operator/pull/1236/commits/80dd4fa2a537c17a981a77d50e237b9b618cf20e